src/scripts/farming/Farming.ts

```diff
@@ -1233,8 +1233,9 @@ class Farming implements Feature {
         }
     }
 
+    public static genBounds = [0, 8, 20, 35, 53, Infinity];
```

Comment:
Do we need the `0` in here?
As then the numbers correctly correlate to the region IDs
```suggestion
    public static genBounds = [8, 20, 35, 53, Infinity];
```